src/sec_newton.mbx

```diff
@@ -558,222 +557,577 @@
+            <var name="$f">
+              <static>\x^4+2x^3-7x^2-x+5</static>
+            </var>
+            <var name="$L">
+              <static>/(-3.714,-0.857,1,1.571)</static>
             </var>
             <pg-code>
-            </pg-code>
+              $f = list_random('x^4+2x^3-7x^2-x+5');
+              $f = Compute($f);
+              $L = List(-3.714,-0.857,1,1.571);
+              Context("LimitedNumeric");
+              </pg-code>
           </setup>
           <statement>
-            <p><m>f(x) = x^4+2x^3-7x^2-x+5</m></p>
+            <p>Use Newton's Method to approximate all roots of <m>f(x)=<var name="$f" /></m> accurate to three places after the decimal. Use technology to obtain good initial approximations. Enter answers in list format, separated by commas.</p>
+            <table>
+              <tabular>
+                <row>
+                  <cell>Roots:</cell>
+                  <cell><var name="$L" width="50"/></cell>
+                </row>
+              </tabular>
+            </table>
           </statement>
-          <solution>
-            <p>roots are: <m>x=-3.714</m>, <m>x=-0.857</m>, <m>x=1</m> and <m>x=1.571</m></p>
-          </solution>
         </webwork>
       </exercise>
+
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$f">
+              <static>\x^17-2x^13-10x^8+10</static>
+            </var>
+            <var name="$L">
+              <static>/(-1.013,0.988,1.393)</static>
+            </var>
+            <var name="$I">
+              <static>/(-2,2)</static>
             </var>
             <pg-code>
+              $f = list_random('x^17-2x^13-10x^8+10');
+              $I = Interval(-2,2);
+              $f = Compute($f);
+              $L = List(-1.013,0.988,1.393);
+              Context("LimitedNumeric");
             </pg-code>
           </setup>
           <statement>
-            <p><m>f(x) = x^{17}-2x^{13}-10x^8+10</m> on <m>(-2,2)</m></p>
+            <p>Use Newton's Method to approximate all roots of <m>f(x)=<var name="$f" /></m> on the interval <m><var name="$I" /></m> accurate to three places after the decimal. Use technology to obtain good initial approximations. Enter answers in list format, separated by commas.</p>
+            <table>
+              <tabular>
+                <row>
+                  <cell>Roots:</cell>
+                  <cell><var name="$L" width="50"/></cell>
+                </row>
+              </tabular>
+            </table>
           </statement>
-          <solution>
-            <p>roots are: <m>x=-1.013</m>, <m>x=0.988</m>, and <m>x=1.393</m></p>
-          </solution>
         </webwork>
       </exercise>
+
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$f">
+              <static>\x^2*cos(x) + (x-1)*sin(x)</static>
```

Comment:
Can implied multiplication always be omitted in both the static and perl sections of code? Also, I assume \cdot is only valid in the static portion.